### PR TITLE
Feature/coach outlet fonts

### PR DESCRIPTION
**Pull request by dmerfeld**

### CODE REVIEW

**Comment by dmerfeld** [comment]
Looks good

**Review by viv-tapestry — APPROVED** [review]
looks good.

**Review by viv-tapestry — APPROVED** [review]
looks good.

**Comment by ryanfitzer** [comment]
@dmerfeld I'm on my phone, so I'm not able to check the build failure log. I expect it's due to the comment I left where you used `2` instead of a valid t-shirt size, therefore breaking the alias.

**Comment by dmerfeld** [comment]
I wasn’t editing sizes, just the font names for coach outlet.

Dan Merfeld | CEO + Founder
TheoryThree Interactive llc
211 S Paterson, Suite #280
Madison, WI 53703
www.theorythree.com
c: 608-575-1066

File: src/brands/coach-outlet/default/font/face.json [hunk 1]
```diff
@@ -36,10 +36,10 @@
             "value": "normal"
           },
           "local": {
-            "value": ["HelveticaLTPro-Bold"]
+            "value": ["HelveticaNeueBoldExtended"]
           },
           "woff2": {
```
Comment (ryanfitzer) [inline src/brands/coach-outlet/default/font/face.json:41]: The property name should match the font extension. Is this font `woff` or `woff2`?

File: src/brands/coach-outlet/default/font/face.json [hunk 2]
```diff
@@ -16,15 +16,15 @@
             "value": "normal"
           },
           "local": {
-            "value": ["HelveticaLTPro-Roman"]
+            "value": ["HelveticaNeueExtended"]
           },
           "woff2": {
```
Comment (ryanfitzer) [inline src/brands/coach-outlet/default/font/face.json:21]: The property name should match the font extension. Is this font `woff` or `woff2`?

**Review by ryanfitzer — CHANGES_REQUESTED** [review]

**Review by ryanfitzer — APPROVED** [review]

**Pull request merged**